src/diagnose.ts

```diff
@@ -273,6 +276,20 @@ export class DiagnoseTool {
     )
   }
 
+  private getCustomClientFilePath() {
+    const flagIndex = process.argv.indexOf("--config")
+
+    if (flagIndex !== -1 && flagIndex + 1 < process.argv.length) {
+      const filePath = process.argv[flagIndex + 1]
+
+      if (path.isAbsolute(filePath)) {
+        return filePath
+      } else {
+        return path.resolve(filePath)
+      }
```

Comment:
I believe resolving an absolute path leaves it unchanged.

```suggestion
      return path.resolve(filePath)
```